hybrid/dispatch/power_sources/pv_dispatch.py

```diff
@@ -22,3 +22,6 @@ def __init__(self,
                  block_set_name: str = 'pv'):
         super().__init__(pyomo_model, indexed_set, system_model, financial_model, block_set_name=block_set_name)
 
+    def update_time_series_parameters(self, start_time: int):
+        self._system_model.value("gen", [max(0, i) for i in self._system_model.value("gen")])  # zero out any load
+        super().update_time_series_parameters(start_time)
```

Comment:
I'm not sure we want to overwrite the gen in the system model. I feel like we still want to run the inverter nighttime load as a grid purchase in the financial model